### PR TITLE
Implementation of Contact Form

### PR DESCRIPTION
**Pull request by BilalLiaquat7**
In this PR: Contact form implement
update style file
update index file
contact svg added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,22 +229,20 @@ <h5>Skills</h5>
...
+
+    <section id="contact">
+        <div class="contact_detail">
+            <h2 class="head">Contact me</h2>
+            <p class="detail">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                I’d love to help with it!
+            </p>
+        </div>
+        <form class="form" action="https://formspree.io/f/mayzgvgj" method="POST">
+            <input class="input" type="text" maxlength="30" name="text" required placeholder="Enter your name">
```
Comment (V-Blaze) [inline index.html:241]: - [ ] You have done great so far, :+1:  but there are a few UI issues you need to fix before moving to the next milestone,
- - According to the project requirement, you are meant to replicate the given design as much as you can, Currently, there is no spacing between your text and the first input field, the spacing is present in the Figma design, I'll suggest you implement this :+1: :100: 
- - Your Current UI:
Screenshot from 2023-06-13 14-39-20

- - Figma design:
Selection_061

File: index.html [hunk 2]
```diff
@@ -229,22 +229,20 @@ <h5>Skills</h5>
...
+            <input class="input" type="email" name="email" required placeholder="Enter your email">
+            <textarea class="input" maxlength="500" name="message" placeholder="Write your message here.." rows="7" required></textarea>
+            <button type="submit" class="submitt" >Get in touch</button>
+        </form>
```
Comment (V-Blaze) [inline index.html:245]: - [ ] You did well using the right button type here. :+1:  and also implement ting the hover state. :+1: But looking at your current implementation, when a user hovers over the button, the text is no more visible, I'll suggest you also change the text color on hover to that the `Get in touch` text can be seen. 

- - Your Button On hover:
Screenshot from 2023-06-13 14-38-58

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @BilalLiaquat7, @ 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized code✔
- Meaningful commit messages👋  ✔

 ## Required Changes ♻️

- [ ] Your project is missing an MIT license file, I'll suggest you create an MIT or LICENSE file and add the correct content, Also remember to link it in your Readme file correctly. You can use [this file ](https://github.com/V-Blaze/news-hub/blob/dev/LICENSE)as a guide: :+1: :smile: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @BilalLiaquat7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**